README.md

```diff
@@ -13,21 +13,21 @@ This repository is home to the `sgdemux` tool for demultiplexing sequencing data
 * [Overview](#overview)
```

Comment:
You may want to view the file itself (right click triple-dots on the right -> view file) to read through and see how it all fits.